packages/rule-components/src/ReportDetails/index.scss

```diff
@@ -1,13 +1,68 @@
+@import '~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_variables';
+
 .ins-c-rule__report-detail {
-  padding: var(--pf-global--gutter--md) var(--pf-global--gutter);
+  padding: var(--pf-global--gutter--md) 40px;
 
   .ins-c-rule__report-detail-link {
     margin-bottom: var(--pf-global--spacer--xs);
     margin-left: var(--pf-global--spacer--form-element);
   }
 
-  .ins-c-rule__risk-detail {
+  .ins-c-rule__risk-detail-title {
     margin-bottom: var(--pf-global--spacer--sm);
+    font-weight: var(--pf-global--FontWeight--light);
+  }
+
+  .ins-c-rule__risk-detail-description-text {
+    color: var(--pf-global--Color--200);
+    size: var(--pf-chart-global--FontSize--sm);
+  }
+
+  .ins-c-rule__link-in-description {
+    display: block;
+    margin-top: var(--pf-l-stack--m-gutter--MarginBottom);
+  }
+
+  .ins-c-rule__risk-description {
+    .ins-c-rule__battery .label {
+      font-weight: bold;
```

Comment:
@Sergey1011010  Font weight for a label should be in this case medium (500)

Reply:
Fixed